### PR TITLE
Renamed the config-main.toml to config.toml. 

### LINKED ISSUE #236: Site does not build with Hugo v0.29 or v0.31
When I run:
```shell
gopheracademy-web$ hugo server
```

No errors are printed, but going to http://localhost:1313/ shows a blank page, specifically:

```shell
$ curl -v http://127.0.0.1:1313/
...
< HTTP/1.1 200 OK
...
<pre>
</pre>
```

When I run `hugo` it generates a bunch of files in the `public` directory, but no HTML files are generated, just lots of index.xml, for example:

```shell
gopheracademy-web$ tree public/
public/
├── README.md
├── advent-2013
│   └── index.xml
├── advent-2014
│   └── index.xml
├── advent-2015
│   └── index.xml
```

If I run `hugo -v` I see errors like:
```
WARN 2017/11/26 13:32:23 [en] Unable to locate layout for "page": [advent-2017/single.html.html advent-2017/single.html _default/single.html.html _default/single.html theme/advent-2017/single.html.html theme/advent-2017/single.html theme/_default/single.html.html theme/_default/single.html]
```

I was running Hugo 0.29 (installed from homebrew), but I updated to 0.31 (the latest version), yet I have the same issues.  I don't know which version people have been using to build the site, but I know Hugo has made breaking changes in the past which have broken sites.

### PR DESCRIPTION
**Pull request by bramp**
The original config.toml doesn't generate a valid site, but is the one used by default. Fixes #236

### CODE REVIEW

**Comment by bketelsen** [comment]
are you sure this needs to be done?
see https://github.com/gopheracademy/gopheracademy-web/issues/236#issuecomment-347223441 for hugo's production configuration.

**Comment by bramp** [comment]
ah sorry yes, race condition between mine and your comments (I didn't see yours before this PR).

Well I don't see how the prod config choses the config-main.toml over the default. But for completeness I'll get a old hugo and see what it does.

But at least something has to be done. Either update the README to say use Hugo 0.15, or tell users to use --config config-main.toml, or this PR.

**Comment by bep** [comment]
I haven't tested this, but I would guess that it is the `layouts-main` config value that is missing in one of those files.

I see no good reason to not use the Hugo default (i.e. `layouts`) for this folder. I would suggest just removing those "special" config options and rename the layouts folder. As it is now, people looking at it will think it is "something special that I don't understand".

**Comment by bramp** [comment]
Ah I'm going to abandon this... So the reason this works in prod, is because the theme is specified on the command line. If I do the same it works, so it does appear hugo is set up to have two different configurations, a config.toml (using gaweb) and config-main.toml using the theme defined in the layouts-main directory.

Coming back I see that https://github.com/gopheracademy/gopheracademy-web/pull/239 is a simple fix to specify the theme in the config, and testing their change it seems to fix my issues.

File: config.toml [hunk 1]
```diff
@@ -2,8 +2,8 @@ baseurl = "https://www.gopheracademy.com"
 languageCode = "en-us"
 title = "Gopher Academy | Home"
 contentdir = "content"
-layoutdir = "layouts-main"
-publishdir = "public-main"
+layoutdir = "layouts"
+publishdir = "public"
```
Comment (bep) [inline config.toml]: I suggest removing those settings entirely. Those are the defaults,  and these will never change in Hugo.

**Comment by bketelsen** [comment]
even better just delete the config-main file   It's not used anymore and causes way too much confusion.  There used to be two sites served from this same repo.

**Pull request closed without merging**